.github/workflows/release-publish.yml

```diff
@@ -98,17 +98,17 @@ jobs:
         with:
           push: ${{ github.event_name == 'release' }}
           tags: |
-            beryju/authentik-${{ matrix.type }}:${{ steps.ev.outputs.version }},
+            beryju/authentik-${{ matrix.type }}:${{ steps.ev.outputs.version }}${{ steps.ev.outputs.versionRC }},
             beryju/authentik-${{ matrix.type }}:${{ steps.ev.outputs.versionFamily }},
-            beryju/authentik-${{ matrix.type }}:latest,
-            ghcr.io/goauthentik/${{ matrix.type }}:${{ steps.ev.outputs.version }},
+            beryju/authentik-${{ matrix.type }}:latest${{ steps.ev.outputs.versionRC }},
+            ghcr.io/goauthentik/${{ matrix.type }}:${{ steps.ev.outputs.version }}${{ steps.ev.outputs.versionRC }},
             ghcr.io/goauthentik/${{ matrix.type }}:${{ steps.ev.outputs.versionFamily }},
-            ghcr.io/goauthentik/${{ matrix.type }}:latest
+            ghcr.io/goauthentik/${{ matrix.type }}:latest${{ steps.ev.outputs.versionRC }}
```

Comment:
This is awful, but I don't know another way, except having the python script generate that list maybe?

Also makes me want to drop support for the `latest` tag.